backend/src/main/java/edu/alexu/cse/dripmeup/service/notifications/AccountManagement.java

```diff
@@ -6,7 +6,7 @@
 import java.io.IOException;
 
 @Service
-public class AccountManagement extends NotificationService {
+public class AccountManagement extends edu.alexu.cse.dripmeup.service.notifications.NotificationService {
```

Comment:
instead of using this long Calling use `import `
`import edu.alexu.cse.dripmeup.service.notifications.NotificationService`